addons/misc/stringtable.xml

```diff
@@ -323,7 +323,7 @@
         </Key>
         <Key ID="STR_kat_misc_SETTING_incompatibilityWarning">
             <English>Incompatible addon warning</English>
-            <German>Inkompatible Mods Warnung</German>
+            <German>Warnung inkompatibler Erweiterungen</German>
```

Comment:
*Inkompatible Mods Warnung
note: using "erweiterung" will just confuse the user and everyone in germany know what a mod is at this point and uses this word as if it was a german word.

Reply:
Changed "Erweiterungen" to "Mods"